packages/competence/bin/static/fragments/frame-work-sites.html

```
@@ -0,0 +1,62 @@
+                    </template>
+                </div>
+            </template>
+
+            <template x-if="isEmpty()">
+                <p class="ti-form-hint" x-text-label="interface.work-sites.empty">No sites configured yet.</p>
+            </template>
+
+            <template x-for="(site, index) in sites" x-bind:key="index">
+                <div class="ti-form-row">
+                    <label class="ti-field-label" x-text-label="interface.work-sites.code">Code</label>
+                    <input class="ti-input" type="text" x-bind:value="site.code" @input="setSiteCode(index, $event.target.value)">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)